order/basic.lean

```diff
@@ -26,8 +26,56 @@ theorem monotone_comp {f : α → β} {g : β → γ} (m_f : monotone f) (m_g :
   monotone (g ∘ f) :=
 assume a b h, m_g (m_f h)
 
+lemma monotone_mul_nonneg {R : Type*} [decidable_linear_ordered_comm_ring R] {a : R} (a_nonneg : 0 ≤ a) : monotone (λ x, a*x) :=
+assume b c b_le_c, mul_le_mul_of_nonneg_left b_le_c a_nonneg
 end monotone
 
+section max
+variable [decidable_linear_order α] 
+
+lemma le_max_of_le_left {a b : α} ( c: α ) (H : a ≤ b ) : a ≤ max b c :=
```

Comment:
there shouldn't be whitespace in type declarations, but around the `:`. In this concrete case `(c : α) (H : a ≤ b)`